### ISSUE TITLE
Options to use mask? 

### ISSUE THREAD

**Issue by kamikazebr** [angular-ui/ui-date#131]
Hi, i was looking for option to mask the date...and not found...

**Comment by denis-bondarenko** [comment]
yeah, i approve the request. a lot of users input dates manually and we should support this scenario
